app/services/auth.js

```diff
@@ -83,6 +83,29 @@ export default Service.extend({
 
   redirectUrl: null,
 
+  init() {
+    this._super(...arguments);
+    window.addEventListener('focus', this.checkAuth.bind(this));
+  },
+
+  checkAuth() {
+    if (!this.currentUser || !this.storage)
+      return;
+    const { accounts } = this.storage;
+    const { vcsId } = this.currentUser;
+    let stillLoggedIn = false;
+
+    accounts.map((user) => {
+      if (user.vcsId === vcsId) {
+        stillLoggedIn = true;
+      }
+    });
+
+    if (!stillLoggedIn) {
+      this.router.transitionTo('signin');
```

Comment:
Are you sure, we don't want to run the full `signOut` with all teardowns and cleanups?

Reply:
This is already performed in another tab - this tab, which signed out before.